remote/github/github.go

```diff
@@ -230,7 +230,7 @@ func (g *Github) Status(u *model.User, r *model.Repo, b *model.Build, link strin
 	status := getStatus(b.Status)
 	desc := getDesc(b.Status)
 	data := github.RepoStatus{
-		Context:     github.String("Drone"),
+		Context:     github.String("continuous-integration/drone"),
```

Comment:
Travis makes a distinction between pull requests and push events, hence the need for a 3rd level, to me they look redundant.